app/models/legislation/draft_version.rb

```diff
@@ -21,6 +28,13 @@ def render_html
 
     self.body_html = Redcarpet::Markdown.new(renderer).render(body)
     self.toc_html = Redcarpet::Markdown.new(toc_renderer).render(body)
+
+    translations.each do |translation|
+      if translation.body_changed?
```

Comment:
Is there a reason why we use the `body_changed?` condition for translations, but we don't use before rendering the body on line 29?

Reply:
No reason I can think of, that's just the way it was and I didn't change it (but I now think I should have, for consistency and to avoid giving the impression that there _is_ a reason for it to be different).